docs/CTXIAOMIPUSH.md

```diff
@@ -39,10 +39,12 @@ Click on [Mi Push Console](http://admin.xmpush.global.xiaomi.com/) and click on
 ## 🚀 Integration
 [(Back to top)](#-table-of-contents)
 
-* Add the CleverTap Xiaomi Push dependency in app’s `build.gradle`
+* Add the CleverTap Xiaomi Push dependency and Mi Push Dependency (from [here](https://admin.xmpush.xiaomi.com/en/mipush/downpage/android)) in app’s `build.gradle`
```

Comment:
@root-ansh-ct I think we should mention specifically that the Mi Push AAR should be added to the libs folder